### ISSUE TITLE
[FEAT]: Adding CogVLM for VQA inference

### ISSUE THREAD

**Issue by Jasonqi146** [lwaekfjlk/mmodel-soup#22]
### Description

CogVLM is a SOTA model on VQA and we need it for two use cases:

1. Making image classification with logits
2. Describe image information into texts
